src/CreatePage/CreatePage.js

```diff
@@ -0,0 +1,323 @@
+import React, { useEffect } from 'react';
+import { Form, Formik, Field, FieldArray } from 'formik';
+import { NavLink } from 'react-router-dom';
+import { TextField } from 'formik-mui';
+import { object, string, array } from 'yup';
+import { useDispatch, useSelector } from 'react-redux';
+import { useAddProjectMutation } from '../api/projectsApi';
+import {
+  createProject,
+  refreshCreateProjectMessage
+} from '../redux/projects/projects-reducer';
+import style from './CreatePage.module.css';
+
+const CreatePage = () => {
+  const dispatch = useDispatch();
+
+  const [addProject] = useAddProjectMutation();
+
+  useEffect(() => {
```

Comment:
You don't need to put stuff into redux if you are using the date inside the same component.